corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -52,6 +52,7 @@ hqDefine('analytix/js/hubspot', [
                         }
 
                         _utils.loadDemoForm(apiId, formId);
+                        _utils.loadTrialForm(apiId, '9c8ecc33-b088-474e-8f4c-1b10fae50c2f');
```

Comment:
Did you mean to hard-code this id here? I don't know much about how hubspot works, but if this is specific to our account, might be better to have it be a setting?

Reply:
yes it's fine as this would be public anyway